helm/values.yaml

```diff
@@ -152,7 +152,7 @@ schemaCompatibility:
     repository: hypertrace/schema-compatibility
     tagOverride: ""
   config:
-    name: "schema-compatibility-configmap"
+    name: "schema-compatibility"
```

Comment:
this shouldn't be residing here, bring it up one level and use as
```
name: {{ .Values.schemaCompatibility.name }}
```
config section should only contain what are used in configmap, anything else shouldn't be under that key, move it level above